### PR TITLE
Changes as per standard

### PR DESCRIPTION
**Pull request by cs-renuka-andhale**
#### Descriptions:
Made the changes as per connector standard.

### CODE REVIEW

File: wiz-io/connector.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+"""
+    Copyright start
+    MIT License
+    Copyright (c) 2023 Fortinet Inc
+    Copyright end
```
Comment (cs-utkarsh-surana) [inline wiz-io/connector.py:5]: Change it to 2024

File: wiz-io/info.json [hunk 2]
```diff
@@ -0,0 +1,216 @@
+{
+  "name": "wiz-io",
+  "version": "1.0.0",
+  "label": "Wiz.io",
+  "description": "Wiz provides a comprehensive analysis engine that integrates: Cloud Security Posture Management (CSPM) Kubernetes Security Posture Management (KSPM) Cloud Workload Protection (CWPP) + vulnerability management. Infrastructure-as-Code (IaC) scanning.",
+  "publisher": "Julian Petersohn",
```
Comment (cs-utkarsh-surana) [inline wiz-io/info.json]: Change it to Fortinet

File: wiz-io/info.json [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+  "icon_small_name": "download.png",
```
Comment (cs-utkarsh-surana) [inline wiz-io/info.json]: Both small and large images are referring to same image

File: wiz-io/info.json [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+      },
+      {
+        "title": "Authentication Endpoint URL",
+        "type": "url",
+        "name": "authenticationEndpointURL",
+        "required": false,
+        "visible": true,
+        "editable": true,
+        "value": "https://auth.app.wiz.io/oauth/token",
+        "tooltip": "URL wich is used to retrieve OAuth Token from. The URL can be found at the Service Account page."
```
Comment (cs-utkarsh-surana) [inline wiz-io/info.json]: Spelling mistakes of 'which' also complete the sentence by specifying the source.

File: wiz-io/info.json [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+      "description": "Get Issues for all Assets from Wiz.io",
+      "parameters": [
+        {
+          "title": "Limit",
+          "type": "integer",
+          "name": "limit",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "value": 100,
```
Comment (cs-utkarsh-surana) [inline wiz-io/info.json]: Description keys are missing in most of the parameter.

File: wiz-io/utils.py [hunk 6]
```diff
@@ -0,0 +1,48 @@
+"""
+This file includes the helper functions for the connector
+"""
+import requests
```
Comment (cs-utkarsh-surana) [inline wiz-io/utils.py:12]: copy right missing

**Review by cs-utkarsh-surana — CHANGES_REQUESTED** [review]

File: wiz-io/playbooks/Sample - Wiz.io - 1.0.0.json [hunk 7]
```diff
@@ -0,0 +1 @@
+null
```
Comment (cs-utkarsh-surana) [inline wiz-io/playbooks/Sample - Wiz.io - 1.0.0.json]: Remove this and add the palybooks file

File: wiz-io/operations.py [hunk 8]
```diff
@@ -0,0 +1,119 @@
+"""
+    Copyright start
+    MIT License
+    Copyright (c) 2023 Fortinet Inc
```
Comment (cs-utkarsh-surana) [inline wiz-io/operations.py]: Change the year

File: wiz-io/info.json [hunk 9]
```diff
@@ -0,0 +1,216 @@
+{
+  "name": "wiz-io",
+  "version": "1.0.0",
+  "label": "Wiz.io",
+  "description": "Wiz provides a comprehensive analysis engine that integrates: Cloud Security Posture Management (CSPM) Kubernetes Security Posture Management (KSPM) Cloud Workload Protection (CWPP) + vulnerability management. Infrastructure-as-Code (IaC) scanning.",
```
Comment (cs-utkarsh-surana) [inline wiz-io/info.json:5]: Add standard keys which are missing eg. cs_approved, cs_compatible

**Review by cs-utkarsh-surana — CHANGES_REQUESTED** [review]

File: wiz-io/info.json [hunk 10]
```diff
@@ -0,0 +1,246 @@
...
+  "publisher": "Fortinet CSE",
+  "cs_approved": true,
+  "cs_compatible": true,
+  "icon_small_name": "small.png",
+  "icon_large_name": "large.png",
+  "category": "Asset Management",
```
Comment (cs-prerna-joshi) [inline wiz-io/info.json]: incorrect category

File: wiz-io/info.json [hunk 11]
```diff
@@ -0,0 +1,246 @@
+{
+  "name": "wiz-io",
+  "version": "1.0.0",
+  "label": "Wiz.io",
+  "description": "Wiz provides a comprehensive analysis engine that integrates: Cloud Security Posture Management (CSPM) Kubernetes Security Posture Management (KSPM) Cloud Workload Protection (CWPP) + vulnerability management. Infrastructure-as-Code (IaC) scanning.",
+  "publisher": "Fortinet CSE",
```
Comment (cs-prerna-joshi) [inline wiz-io/info.json]:
```suggestion
  "publisher": "Fortinet",
```

File: wiz-io/info.json (hunk 10 above)
Comment (cs-renuka-andhale) [reply wiz-io/info.json]: Please suggest category ?

**Pull request merged**